python/tvm/script/builder/tir/for_frame.py

```diff
@@ -29,13 +29,17 @@
 class ForFrame(TIRFrame):
     def __enter__(self) -> List[Var]:
         _base_ffi_api.FrameEnter(self)  # pylint: disable=no-member # type: ignore
-        return self.vars
+        return self.vars if len(self.vars) > 1 else self.vars[0]
 
 
 def serial(start, stop, annotations=None) -> ForFrame:
     return _ffi_api.Serial(start, stop, annotations)  # pylint: disable=no-member # type: ignore
 
 
+def serial(stop, annotations=None) -> ForFrame:
+    return _ffi_api.Serial(0, stop, annotations)  # pylint: disable=no-member # type: ignore
```

Comment:
This fix is not correct. Note that python doesn't actually support function overloading, and this `serial` method will override the previous one in line 35-36, which means users cannot write `serial(start=0, stop=10, annotations=None)` after this fix